### ISSUE TITLE
chore(docs): Update CSS Blocks Readme to be more beginner friendly.

### ISSUE THREAD

**Issue by abritinthebay** [linkedin/css-blocks#410]
The current readme has a lot of great content but it's kind of an info-dump. 

Let's turn it into a more concise readme for the project: one that provides a good understanding of how to get started and how the project is organized. It can then link out to other, more detailed, docs when required. Newbie friendly!

**Comment by abritinthebay** [comment]
Proposed Outline:

- Intro
- Links to template integrations and build system integrations
- Getting started (mono-repo, each project has own docs, etc)
- Quick feature overview (high level, no detail, links to more in-depth descriptions like static analysis)
- Super quick syntax example (with link to more complex one)
- Summary of packages (by type - core/ template / build / public sites)
- Suggested reading (architecture, etc)

</fin>

long story short: moving a lot of the content out into docs and linking to it so it's easier to read the readme and find specific docs.
